app/Services/GitHub/GitHubClientFactory.php

```
@@ -10,23 +11,84 @@
+
+    /**
+     * @return array{0:string,1:?string,2:string}|null
+     */
+    protected function resolveAuthenticationPayload(array $config): ?array
+    {
+        $method = strtolower($config['method'] ?? 'token');
+
+        return match ($method) {
+            'none', 'anonymous' => null,
+            'token', 'access_token' => $this->payloadWith($config, ['token'], $method, Client::AUTH_ACCESS_TOKEN),
+            'password', 'basic' => $this->payloadWith($config, ['username', 'password'], $method, Client::AUTH_HTTP_PASSWORD),
```

PHPStan (2.1.31)

Access to undefined constant Github\Client::AUTH_HTTP_PASSWORD.


(classConstant.notFound)